content/sql/concepts/window-functions/terms/NTILE/NTILE.md

```diff
@@ -0,0 +1,84 @@
+---
+Title: 'NTILE()'
+Description: 'Groups data together from a window function in roughly equal groups.'
+Subjects: 
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'SQL'
+  - 'MySQL'
+  - 'PostgreSQL'
+  - 'SQLite'
+  - 'Functions'
+CatalogContent:
+  - paths/analyze-data-with-sql
+---
+
+The **`NTILE()`** SQL function groups data into roughly equal groups like the SQL "GROUP BY" clause. However `NTILE()` is applicable to window functions. A number can be specified with the function which will display the data organized roughly across a specified number of groups.
+
+## Syntax
+
+The `...` used below is meant to indicate code before or after the `NTILE()` clause below. It's not part of the actual syntax.
+
+A number is required between the parenthesis.
+
+An `ORDER BY` and a `PARTITION BY` clause is a good idea to include. It makes the data more organized. The `ORDER BY` SQL clause can show query results in ascending or descending order. They can be specified with `ASC` or `DESC` respectively.
+
+```pseudo
+  ... 
+  NTILE(number_of_groups_to_split_data_between) OVER (
+    PARTITION BY column1_name
+    ORDER BY column2_name DESC
+  ) ...
+```
+
+## Example
+
+Let's say there is a "basketball_points" table where the first 10 rows look like this:
+
+| player | Week | total_points |
```

Comment:
```suggestion
| player | week | total_points |
```

Reply:
For consistency I thought it would be better to have this lowercase to match the rest of the headers